cfg/builder/builder_walk.cc

```diff
@@ -451,25 +451,24 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::ExpressionPtr &what, BasicBlo
                 auto aliasName = cctx.newTemporary(core::Names::cfgAlias());
                 auto loc = a.loc();
 
-                if (a.symbol() == core::Symbols::StubModule()) {
+                if (auto sym = a.symbol(); sym == core::Symbols::StubModule()) {
```

Comment:
Preference for doing this assignment outside the `if` condition if we're explicitly attempting to read it in the `else` branch
```suggestion
                auto sym = a.symbol();
                if (sym == core::Symbols::StubModule()) {
```